portal-ui/src/screens/Console/Groups/Groups.tsx

```diff
@@ -106,6 +106,26 @@ const Groups = ({ classes, setErrorSnackMessage, history }: IGroupsProps) => {
     IAM_SCOPES.ADMIN_GET_GROUP,
   ]);
 
+  const selectionChanged = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const targetD = e.target;
```

Comment:
this could be simplified using obhect destructuring 
```suggestion
    const { target:{ value, checked } ={} } = e 
    
```